### ISSUE TITLE
【2018】The Lovasz-Softmax loss: A tractable surrogate for the optimization of the ´ intersection-over-union measure in neural networks

### ISSUE THREAD

**Issue by sanoyo** [sanoyo/all-paper-deep-learning-machine-lerning#41]
## 要約
セグメンテーションに使用できる新しいloss関数を提案しており、
セグメンテーションでよく使用される `cross entropy loss`よりも良い結果を出している。

離散値であるlossをなめらかな連続空間で表現でいるようにしている。(lovast-extension)

![image](https://user-images.githubusercontent.com/24674237/53883019-3eb8a280-405b-11e9-8425-aa10a2c9046d.png)

## 論文リンク
https://arxiv.org/pdf/1705.08790.pdf

## 参考にした記事
https://knto-h.hatenablog.com/entry/2018/10/09/170000
